src/Features/Admin/Users/EditUser/index.js

```diff
@@ -77,10 +104,46 @@ const AddUser = () => {
     },
   ]);
 
+  const userById = FetchUserById(id);
+
+  useEffect(() => {
+    if (userById !== undefined) {
+      setUser(userById);
+      fields.map((field) => {
+        if (field.label === 'Email') {
+          field.value = userById.email;
+        } else if (field.label === 'Name') {
+          field.value = userById.name;
+        } else if (field.label === 'Password') {
+          field.value = userById.password;
```

Comment:
can't we do it like `filed.value = useById[field.label.toLowerCase()]; `???
So we can avoid all those if else